GW2EIEvtcParser/ParserHelpers/ArcDPSEnums.cs

```diff
@@ -764,6 +764,7 @@ public enum TrashID : int
             // Ensolyss
             NightmareHallucination1 = 16912, // (exploding after jump and charging in last phase)
             NightmareHallucination2 = 17033, // (small adds, last phase)
+            NightmareAltar = 35791,
```

Comment:
That looks like a GadgetID, unlikely to be reliable as they tend to be volatile.